reps/config.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = {
+  rootPath: "../../mozilla-central",
+  devtoolsPath: "../../mozilla-central/devtools",
```

Comment:
Why did you change the default path?
The recommended directory structure is described here:
https://github.com/janodvarko/prototypes/tree/master/reps#source-directory-structure

And looks like as follows:

<root>
  - mozilla.org
    - fx-team
      + devtools
  - github.com
    - janodvarko
      - prototypes
        + reps

For this to work there should be:
`../../../../mozilla.org/`
`../../../../mozilla.org/fx-team/devtools`

Honza